### PR TITLE
HilNetworkDriver implementation

### PR DESCRIPTION
**Pull request by vsemp**
This is HilNetworkDriver implementations. It uses fuctions from hilapi.py and from HilInventoryDriver. Since HilInventoryDriver the following functions relying on HilInventoryDriver are also not implemented: __read_current_state(self, argv) and __write_current_state(self, argv)

### CODE REVIEW

File: lib/hardware_services/network_drivers/HilNetworkDriver.py [hunk 1]
```diff
@@ -14,19 +14,74 @@
...
+
+            time.sleep(1)
+            hilapi.node_connect_network(host_to_move,
+                                nic_json['label'],
+                                new_cloud,
+                                'null')
+            time.sleep(1)
+
+        node_nics = hilapi.show_node(host_to_move)['nics']
+        print("TESTING: After move " + str(node_nics))
```
Comment (portante) [inline lib/hardware_services/network_drivers/HilNetworkDriver.py]: Perhaps put this under a debug conditional, like: `if _DEBUG > 0:` and then have a module global you can play with if necessary.

File: lib/hardware_services/network_drivers/HilNetworkDriver.py [hunk 2]
```diff
@@ -14,19 +14,74 @@
...
+    def __write_current_state(self, argv):
+        """
+        Writes/updates information about a new network of a node to the HIL database
+        """
+        
+        pass
+        
+    def __move_one_host(self, argv):
+        if len(argv) != 3:
+            sys.exit("Incorrect number of arguments. Should be host_to_move old_cloud new_cloud.")
```
Comment (portante) [inline lib/hardware_services/network_drivers/HilNetworkDriver.py]: Call `sys.exit` in a library makes it difficult to know how to use the library in different situations.  Instead, raise and exception.

However, if `__move_one_host` is only called within this module, why not just explicitly list all the arguments?   Python has variable length argument support built in, so using a list is not considered "pythonic".

File: lib/hardware_services/network_drivers/MockNetworkDriver.py [hunk 3]
```diff
@@ -10,25 +10,14 @@
 import logging
 from subprocess import call
 from subprocess import check_call
+from os import path
+
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```
Comment (portante) [inline lib/hardware_services/network_drivers/MockNetworkDriver.py]: Why do we need this?  Won't this be adding `lib/` to the class path?

File: lib/hardware_services/network_drivers/QuadsNativeNetworkDriver.py [hunk 4]
```diff
@@ -11,6 +11,9 @@
 import logging
 from subprocess import call
 from subprocess import check_call
+from os import path
+
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
```
Comment (portante) [inline lib/hardware_services/network_drivers/QuadsNativeNetworkDriver.py]: Why do we need this?  Won't this be adding `lib/` to the class path?

File: lib/hardware_services/network_drivers/HilNetworkDriver.py [hunk 5]
```diff
@@ -14,19 +14,74 @@
 from subprocess import call
 from subprocess import check_call
 
-from hardware_services.network_service import NetworkService
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "util"))
```
Comment (portante) [inline lib/hardware_services/network_drivers/HilNetworkDriver.py]: Do we need this?  If we remove this line ...

File: lib/hardware_services/network_drivers/HilNetworkDriver.py [hunk 6]
```diff
@@ -14,19 +14,74 @@
...
 
-# added for EC528 HIL-QUADS integration project
-hil_url = 'http://127.0.0.1:5000'
+import hilapi
```
Comment (portante) [inline lib/hardware_services/network_drivers/HilNetworkDriver.py]: And change this to `import lib.hardward_services.utils.hilapi`, where we add a __init__.py module to the `utils` directory, won't we avoid appending this to the path?

**Review by portante — CHANGES_REQUESTED** [review]

File: lib/hardware_services/network_drivers/QuadsNativeNetworkDriver.py (hunk 4 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/QuadsNativeNetworkDriver.py]: Yes. I took it from
https://github.com/BUEC528-HIL-QUADS/quads/blob/master/lib/hardware_services/inventory_drivers/HilInventoryDriver.py

File: lib/hardware_services/network_drivers/HilNetworkDriver.py (hunk 6 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: Most likely, I don't quiet know how Python sys.paths work.

Is the path to ......./quads/ added by default somehow?

File: lib/hardware_services/network_drivers/HilNetworkDriver.py (hunk 1 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: Will do this.

File: lib/hardware_services/network_drivers/HilNetworkDriver.py (hunk 2 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: Will do this.

File: lib/hardware_services/network_drivers/MockNetworkDriver.py (hunk 3 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/MockNetworkDriver.py]: Is lib/ already added? Why?

File: lib/hardware_services/network_drivers/HilNetworkDriver.py (hunk 5 above)
Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: Would you be so kind to briefly explain why we don't need it?

You can just give me a link. @portante

Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: I know we can do 
```
from . import echo
from .. import formats
from ..filters import equalizer
```

Comment (vsemp) [reply lib/hardware_services/network_drivers/HilNetworkDriver.py]: Is this what you want?

**Pull request merged**